common/config/config.json

```diff
@@ -5,5 +5,6 @@
   },
   "storage": "ipfs",
   "ipfsGateway": "https://ipfs.io/ipfs/",
-  "swarmGateway": "https://api.gateway.ethswarm.org/bzz/"
-}
+  "swarmGateway": "https://api.gateway.ethswarm.org/bzz/",
+  "swarmServer": "https://api.gateway.ethswarm.org"
```

Comment:
use only one of these 2 , swarmServer typically, and deduce the other one (with a function or not)

Reply:
done